templates/renaissance/home.html.twig

```diff
@@ -60,18 +58,14 @@
             <div
                     class="md:sticky md:top-8 flex-1 md:w-1/3 border-t border-gray pt-7.5 md:pt-12.5"
             >
-                <p
-                        class="text-light-blue font-bold text-[1.5rem] leading-8 md:text-custom-xl mb-6.25 md:mb-7.5"
-                >
+                <p class="text-dark-blue font-bold text-[1.5rem] leading-8 md:text-custom-xl mb-6.25 md:mb-7.5">
                     Renaissance: <br/>
-                    Qui <span class="stabilo stabilo-rose">sommes-nous?</span>
+                    Qui <span class="stabilo stabilo-jaune">sommes-nous?</span>
```

Comment:
We don't have the image png for this stabilo
```suggestion
                    Qui <span class="stabilo stabilo-yellow">sommes-nous?</span>
```

Reply:
I know, we need to add it